### PR TITLE
Feature/update jenkins build

### LINKED ISSUE #406: Update jenkins build
Jenkins build should use scripts from repo not inlined ones

### PR DESCRIPTION
**Pull request by aju**
closes #406
Now Jenkins sets env variables and runs `deploy/ci.sh`

### CODE REVIEW

File: deploy/ci.sh [hunk 1]
```diff
@@ -1,10 +1,12 @@
 #!/bin/bash
 set -e
 
-./scripts/cibuild.sh
+export IMAGE_ARTIFACT=${REGISTRY}/${JOB_NAME}:${BUILD_NUMBER}-${GIT_SHORT_COMMIT}
+export PROJECT_DIR=$(pwd)
+
+echo "IMAGE_ARTIFACT=$IMAGE_ARTIFACT"
```
Comment (kdzwinel) [inline deploy/ci.sh]: console.log? Do we need that?

**Comment by kdzwinel** [comment]
ok

**Comment by mr-mig** [comment]
The problem right now is that you cannot build the project on CI separately from deploy. 
It violates https://github.com/brainly/documents/blob/master/rfc/scripts-pattern.md

**Comment by aju** [comment]
@mr-mig updated

**Comment by mr-mig** [comment]
:+1:

**Pull request merged**